### PR TITLE
Add schedule

### PR DESCRIPTION
**Pull request by misslivirose**
This PR adds the schedule for the Responsible AI challenge to the builders challenge page.

### CODE REVIEW

File: pages/builders-challenge/index.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h2>by Mozilla Builders</h2>
 <section class="mzp-l-content mzp-t-content-xl c-challenge-event">
     <p class="c-event-tagline">AI is changing everything. Now is the time to develop it responsibly. <br> Join us in the Responsible AI Challenge and help us shape the future of AI.</p>
     <p class="c-event-date">In-person event: May 31, 2023 in San Francisco, CA</p>
-    <p>Applications have closed. Thank you for your interest and participation</p>
+    <img src="/img/builders-challenge/AI-Challenge-schedule.png"></img>
```
Comment (pmac) [inline pages/builders-challenge/index.html]:
```suggestion
    <img src="/img/builders-challenge/AI-Challenge-schedule.png">
```
Also missing the required `alt` attribute.

Comment (pmac) [reply pages/builders-challenge/index.html]: Using an image, especially one with no alt text explanation is not accessible. This isn't a ton of info. Can we code this into a table on the page so people using screen readers can also access the information?

**Review by pmac — CHANGES_REQUESTED** [review]

File: pages/builders-challenge/index.html (hunk 1 above)
Comment (pmac) [reply pages/builders-challenge/index.html]: Or maybe the image could be a link to a properly made PDF?

**Comment by pmac** [comment]
There are also a lot of unrelated commits and a merge commit in the git log of this PR. That will need to be cleaned up before we can merge.

**Comment by misslivirose** [comment]
Thanks @pmac! I changed the image to a table and squashed the commits on a new branch - closing this PR in favor of #55.

**Pull request closed without merging**